relayer/chains/evm/provider.go

```diff
@@ -203,7 +231,10 @@ func (cp *EVMChainProvider) RelayPacket(ctx context.Context, packet *bandtypes.P
 			log.Error("saveTransaction error", "retry_count", retryCount, err)
 		}
 
-		txResult := cp.WaitForTx(ctx, txHash, log)
+		txResult, err := cp.WaitForConfirmedTx(ctx, txHash, log)
```

Comment:
it should not be used for both error and txResult. think we should add error detail to txresult instead for this case.